tests/scaffoldings/addons_env_double/custom/src/repos.yaml

```
@@ -0,0 +1,20 @@
+# Odoo is always required
+./odoo:
+  defaults:
+    # Shallow repositores are faster & thinner
+    depth: $DEPTH_DEFAULT
+  remotes:
+    ocb: https://github.com/OCA/OCB.git
+    odoo: https://github.com/odoo/odoo.git
+  target: ocb $ODOO_VERSION
+  merges:
+    - ocb $ODOO_VERSION
+
+rma-old:
+  defaults:
+    depth: $DEPTH_MERGE
+  remotes:
+    origin: https://github.com/OCA/rma.git
+  target: origin 242b7e21a174e48f713459f836794560e68290de # should be before commit 96d2081987292ae02efda9826cbe5ba1ad84d6c3
```

The comment is not much helpful... Also, `target` must be a branch name:

```suggestion
  target: origin _merged_branch
```